src/app/app.routes.ts

```diff
@@ -27,6 +28,14 @@ export const routes: Routes = [
     component: TodoItemComponent,
     canActivate: [AuthGuard],
   },
-  { path: 'app-login', component: LoginComponent },
-  { path: 'app-signup', component: SignupComponent },
+  {
+    path: 'app-login',
```

Comment:
remove the app-, so the route would be a bit more logical /login instead of /app-login same for sign up